project-birth/CONTRIBUTING-DURING-START.md

```diff
@@ -76,7 +76,23 @@ When rebranding source code that creates or displays strings the end user will s
 
 A goal for the core of the editor is to 'decaffeinate' that is to remove CoffeeScript and replace it with JavaScript. This comes largely as a decision since very few of our core team are familiar with or prefer CoffeeScript over JavaScript. Additionally a big push of `Pulsar` is to lower the barrier to entry for new contributors to be as low as possible. And having fewer languages in use assists in that goal. 
 
-As for how to successfully Decaffeinate the core of the editor, a great template for successfully doing so can be seen on [`pulsar-edit/pulsar` PR #13](https://github.com/pulsar-edit/pulsar/pull/13). This PR initially used [online decaffeination services](https://decaffeinate-project.org/repl/), then went through the source code using modern variable declarations by changing `var` to `const` or `let`. Additionally unneeded nested ternary operators were replaced with If statements. From there, that alone may be enough to successfully decaffeinate a file, although it can always be taken further as viewed in the above linked PR.
+As for how to successfully Decaffeinate the core of the editor (or any other repos/packages in use), a great template for successfully doing so can be seen on [`pulsar-edit/pulsar` PR #13](https://github.com/pulsar-edit/pulsar/pull/13). 
+
+What you must keep in mind is that when Atom was originally created, they mostly used tooling for CoffeeScript 1.x whereas much of the tooling that's available now is CoffeeScript 2.x based. This means that in order to decaffeinate, CoffeeScript made for 1.x, without breaking changes, the tooling for version 1.x **MUST** be used otherwise there may be unknown ramifications. 
+
+But with that said its important to note that not all of Atom's code is CoffeeScript 1.x based. That is to say there is a mix of both used throughout the codebase. In order to accurately determine which is being used within the repo you are working on the best way to do so is to look at that repo's `package.json` `devDependencies` or sometimes it's `dependencies`.
+
+Within those fields look for the `coffee-script` or `coffeescript` package to see if its versions are 1.x or 2.x.
+
+In generally it'll probably be 1.x tooling used. 
```

Comment:
This is the only part I'm probably going to mention and maybe right after. This should be "In general" combined with the next part "Now that the version is known,". It just seems odd to me how it's worded here.